src/services/__tests__/product.test.ts

```diff
@@ -79,6 +79,7 @@ describe("Products service", () => {
         tag: "error",
         message,
         errors,
+        globalErrors: [],
```

Comment:
I wonder if we should use the objectContaining matcher here and remove the globalError:
`
expect(response).toEqual({
       expect.objectContaining({
        tag: "error",
        message,
        errors
     }})
`
instead of adding it for every test. Because, if backend adds another field in the future, we have to touch all tests again, right? and we don't test the global error in this test here. 
But I'm not sure what's cleaner, up to you

Reply:
Yes but I think is better for the future to be explicit, and I forgot to test the globalErrors for the warranty